lib/api/core/models/security/profile.js

```diff
@@ -21,23 +38,33 @@ function Profile () {
 Profile.prototype.isActionAllowed = function (requestObject, context, kuzzle) {
   var
     deferred = q.defer();
-
   if (this.roles === undefined || this.roles.length === 0) {
     return q(false);
   }
 
-  async.some(this.roles, (role, callback) => {
-    role.isActionAllowed(requestObject, context, kuzzle)
-      .then(isAllowed => {
-        callback(isAllowed);
+  this.getRoles(kuzzle)
+    .then(roles => {
+      async.some(roles, (role, callback) => {
+        role.isActionAllowed(requestObject, context, kuzzle)
```

Comment:
We still user async v1.5.2. 
And callback arguments for async.some have changed in 2.0.0, with the node-js style: 
https://www.npmjs.com/package/async#some 

> Callback arguments changed in 2.0

In our actual version, it's not possible to use nodeify. We should probably upgrade to v2.0.0-rc6 (but we have to check if we have no regression)... or just add the error catching here.